types/pleasejs/index.d.ts

```diff
@@ -16,11 +16,20 @@ declare namespace PleaseJS{
 
         /***
          * make a color scheme
+         * @param {HSV} base_color
          * @param {MakeSchemeOption} options
          * @returns {Array}
          */
         make_scheme(base_color: HSV, options?: MakeSchemeOption): Array<string | RGB | HSV>;
 
+        /***
+         * make a contrasting color scheme
+         * @param {HSV} base_color
+         * @param {MakeContrastOption} options
+         * @returns {string|RGB|HSV}
```

Comment:
```suggestion
         * @returns {string | RGB | HSV}
```
same format as in the code